ngraph/core/include/ngraph/op/constant.hpp

```diff
@@ -307,6 +308,11 @@ namespace ngraph
                     return rc;
                 }
 
+                std::shared_ptr<runtime::AlignedBuffer> get_aligned_buffer() const
```

Comment:
Why do you need to introduce new method only for test usage?

I think in tests you can use `get_data_ptr()` to compare pointers.

Reply:
Done.